app/views/admin/loans/content/_breakeven_table_report.html.slim

```diff
@@ -19,12 +19,15 @@
         tr.group-content
           td = item[:name]
           td.text-right = item[:quantity]
-          td.text-right = number_to_currency(item[:amount])
-          td.text-right = number_to_currency(item[:total])
+          td.text-right 
+            = @loan.ensure_currency.format_amount(item[:amount], tooltip: false)
```

Comment:
I think we should be handling this using a presenter instead of putting HTML into a model. This also violates Law of Demeter currently. 

We don't have presenters in this project but I think it's time we go there. I like the approach here: http://nithinbekal.com/posts/rails-presenters/ 

So I think this would be `= present(@loan.currency).format_amount(item[:amount], tooltip: false)`

Or if you'd rather my second choice would be to use good old helpers, so

`= format_currency(@loan.currency, item[:amount], tooltip: false)`